### DISCUSSION TITLE
React JS goes blank when installing @prisma/client

### DISCUSSION THREAD

**Discussion by dxkyy** [prisma/prisma#15404] [Q&A]
### Bug description

I made a basic input form using react.js to save stuff to my database, but once I installed `@prisma/client`, my whole react app went blank, how can I fix this?

### How to reproduce

1. Create a basic react.js project
2. setup Prisma for PostgreSQL
3. install `@prisma/client`
5. See blank page

### Prisma information

```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = [my url]
}

model populus {
  id          String   @id
  firstName   String
  lastName    String
  birthday    String
  phoneNumber String
}

```

```js
import { customAlphabet } from "nanoid";
const nanoid = customAlphabet("1234567890", 10);

const savedPerson= await prisma.populus.create({
    data: {
      id: nanoid(4),
      firstName: firstName,
      lastName: lastName,
      birthday: birthday,
      phoneNumber: phoneNumber,
    },
  });
console.log(savedPerson)
```

### Environment & setup

- OS: Windows 10
- Database: PostgreSQL
- Node.js version: 18.9.0

### Prisma Version

```
Environment variables loaded from .env
prisma                  : 4.3.1
@prisma/client          : 4.3.1
Current platform        : windows
Query Engine (Node-API) : libquery-engine c875e43600dfe042452e0b868f7a48b817b9640b (at ..\..\..\..\AppData\Roaming\npm\node_modules\prisma\node_modules\@prisma\engines\query_engine-windows.dll.node)
Migration Engine        : migration-engine-cli c875e43600dfe042452e0b868f7a48b817b9640b (at ..\..\..\..\AppData\Roaming\npm\node_modules\prisma\node_modules\@prisma\engines\migration-engine-windows.exe)
Introspection Engine    : introspection-core c875e43600dfe042452e0b868f7a48b817b9640b (at ..\..\..\..\AppData\Roaming\npm\node_modules\prisma\node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary           : prisma-fmt c875e43600dfe042452e0b868f7a48b817b9640b (at ..\..\..\..\AppData\Roaming\npm\node_modules\prisma\node_modules\@prisma\engines\prisma-fmt-windows.exe)
Format Wasm             : @prisma/prisma-fmt-wasm 4.3.0-32.c875e43600dfe042452e0b868f7a48b817b9640b
Default Engines Hash    : c875e43600dfe042452e0b868f7a48b817b9640b
Studio                  : 0.473.0

```

**Comment by austincrim** [comment]
Hey @dxkyy 👋,

Could you provide a minimal reproduction repository so that I can recreate this issue? There are a lot of things that could be at play here, a repo is the best way forward.

**Reply by vladi-stevanovic** [reply]
👋 Hello @dxkyy - did you perhaps have a chance to work on a minimal reproduction repository? We'd love to continue investigating this so that we can properly log it as a bug!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
